extension-config.json

```diff
@@ -0,0 +1,45 @@
+{
+  "version": "1.0.0",
+  "id": "@shopgate/ext-terms-and-conditions",
+  "components": [
+    {
+      "id": "TermsAndConditions",
+      "path": "frontend/portals/PaymentBarBefore",
+      "target": "cart.item-list.after",
+      "type": "portals"
+    },
+    {
+      "id": "CheckoutButton",
+      "path": "frontend/portals/CheckoutButton",
+      "target": "cart.checkout-button",
+      "type": "portals"
+    },
+    {
+      "id": "isOrderableByTerms",
+      "path": "frontend/reducers/index.js",
+      "type": "reducers"
+    }
+  ],
+  "configuration": {
+    "checkBoxValues": {
+      "type": "admin",
+      "destination": "frontend",
+      "default": ["Value 1", "Value 2", "Value 3"],
```

Comment:
I'd suggest use different schema for the config so it's easy to extend it later:
[
{text: "Text that will be shown" }
]

Later we will extend it with optional/required config for each checkbox.

Reply:
Updated to: 
        {
          "text": "Value 1",
          "textColor": "#000"
        }